Userland/Libraries/LibCore/Stream.h

```diff
@@ -72,7 +72,7 @@ class SeekableStream : public Stream {
 public:
     /// Seeks to the given position in the given mode. Returns either the
     /// current position of the file, or an errno in the case of an error.
-    virtual ErrorOr<off_t> seek(i64 offset, SeekMode) = 0;
+    virtual ErrorOr<off_t> seek(i64 offset, SeekMode = SeekMode::SetPosition) = 0;
```

Comment:
Note that default arguments on virtual functions are a little unruly, now either of these has to happen:
- all accesses to any of the overriding `seek()` functions must either go through this specific one (i.e. `SeekableStream::seek`) or provide all arguments
- all the classes inheriting from this class and overriding this function must provide *the same default*

otherwise it would get extremely confusing and weird.

Reply:
I think I'll move that down to the individual non-virtual functions, and remove it from here.

Reply:
I still don't think that's a great idea, because it's error prone. Sure, you could just say that all functions must take `SetPosition` by default, but it's hard to actually enforce that. I say let the user of the function decide what they want to pass in, it's only a few keystrokes.

Reply:
Ok, it's gone now